.travis.yml

```diff
@@ -2,6 +2,11 @@ language: php
 
 sudo: false
 
+addons:
+    apt:
+        packages:
+            - devscripts
```

Comment:
:-1:

Reply:
@keradus I found a more recent version of `checkbashisms`. Its home page is located here: https://sourceforge.net/projects/checkbaskisms/. `checkbashisms` is just a Perl script that can be downloaded using `wget` and put into `/usr/bin`. How would you recommend me to install it for CI?

Reply:
either install `checkbashisms` as apt, similar to like you do it now, or manually. both are fine for me, as long as you do it only for the only job that needs it.
Currently, you install it again and again for every single job we have.